### ISSUE TITLE
IrfanView settings reset and cannot be modified

### ISSUE THREAD

**Issue by arisudesu** [microsoft/winget-pkgs#14038]
# Brief description of your issue
Just did `winget upgrade IrfanView` from 4.56 to 4.57, after launch the app says it cannot write config INI file:

![изображение](https://user-images.githubusercontent.com/6342849/119797418-e479b580-bee2-11eb-8b72-70dfe401cf74.png)

It should not happen, because the interactive installer has the option to configure this:
![изображение](https://user-images.githubusercontent.com/6342849/119797761-30c4f580-bee3-11eb-9330-a73bb3dcf16e.png)

Seems to me that the first option is selected when installed with winget. It shouldn't be so, we are living in 2020s, not in 90s and apps shouldn't write to their install dir. Or at least, why doesn't it use VIrtualStore?

# Expected behavior
App runs correctly with user settings saved in user profile.

# Actual behavior
App cannot write its INI file because it is located in ProgramFiles.

# Environment
winget v1.0.11451, installed from appx.

**Comment by OfficialEsco** [comment]
I think this is a issue between AllUsers and CurrentUser install, i've added the user install in IrfanView 4.58 #14037

**Comment by OfficialEsco** [comment]
Please test if it works for you then close the issue 🙂

**Comment by lychichem** [comment]
It may also relates to install switch, which automatically select the first option during silent install. Additional switch(es) should be add to select the second option.

**Comment by OfficialEsco** [comment]
> It may also relates to install switch, which automatically select the first option during silent install. Additional switch(es) should be added to select the second option.

`/desktop=1 /thumbs=0 /group=1 /allusers=0 or 1 /assoc=0` is already in the manifest `/allusers=0` seems to select User on both, do you know of another switch?

**Comment by arisudesu** [comment]
Settings file is ok now, though in process of uninstall then install, user settings were removed. Idk if it is normal behavior or not.

**Comment by lychichem** [comment]
> > It may also relates to install switch, which automatically select the first option during silent install. Additional switch(es) should be added to select the second option.
> 
> 
> 
> `/desktop=1 /thumbs=0 /group=1 /allusers=0 or 1 /assoc=0` is already in the manifest `/allusers=0` seems to select User on both, do you know of another switch?

No, unless I can access the script used to build the installer, I am unable to know another switch to control the selection to use the former user setting.

**Comment by iDolmatov** [comment]
Missed switcher to ini path.
https://www.irfanview.com/faq.htm

> iview452_setup.exe /silent /ini=^%APPDATA^%\irfanview
>iview452_setup.exe /silent /ini="%APPDATA%\irfanview"

**Comment by denelon** [comment]
Is this issue resolved?

**Comment by iDolmatov** [comment]
https://github.com/microsoft/winget-pkgs/blob/master/manifests/i/IrfanSkiljan/IrfanView/4.60/IrfanSkiljan.IrfanView.installer.yaml

```
InstallerSwitches:
  Silent: /silent
  SilentWithProgress: /silent
```

Is the INI parameter visible?

**Comment by OfficialEsco** [comment]
https://github.com/microsoft/winget-pkgs/blob/bf2ac60e3d5fda6b8a20701ca28f7a20b67bdda2/manifests/i/IrfanSkiljan/IrfanView/4.60/IrfanSkiljan.IrfanView.installer.yaml#L13-L15
https://github.com/microsoft/winget-pkgs/blob/bf2ac60e3d5fda6b8a20701ca28f7a20b67bdda2/manifests/i/IrfanSkiljan/IrfanView/4.60/IrfanSkiljan.IrfanView.installer.yaml#L113-L125

How would `/ini=^%APPDATA^%\irfanview` work on a shared computer?
